kong/llm/init.lua

```diff
@@ -239,6 +270,104 @@ local function identify_request(request)
   end
 end
 
+local function get_token_text(event_t)
+  -- chat
+  return
+    event_t and
+    event_t.choices and
+    #event_t.choices > 0 and
+    event_t.choices[1].delta and
+    event_t.choices[1].delta.content
+
+    or
+
+  -- completions
+    event_t and
+    event_t.choices and
+    #event_t.choices > 0 and
+    event_t.choices[1].text
+
+    or ""
+end
+
+function _M:calculate_cost(query_body, tokens_models, tokens_factor, tokens_max_query_cost)
```

Comment:
nit: looks like there's some inconsistent indentation (4 spaces vs 2) in here.

Reply:
It's because it's under the "return" statement indent : )

Reply:
What I'm seeing is that `_M:calculate_cost()` and `local count_prompt()` use 4 spaces for indentation, whereas most of the file uses 2.

Reply:
Oh yeah :S c31d7bd56f404363683dfb3885c2de6a0c584ce9